syft/pkg/cataloger/golang/test-fixtures/archs/README.md

```diff
@@ -0,0 +1,4 @@
+The assets in this folder come from two places:
+
+* Golang's internal test data
```

Comment:
this isn't true within the `archs` directory, but is true within the `test-fixtures` directory, right?